kafka-to-log/README.md

```diff
@@ -7,12 +7,12 @@ This example shows how to read a Kafka topic, using the camel `kafka` component,
 2. A topic named `test-topic` with some messages in it
 
 ## How to run
-1. Open the `kafka.camel.yaml` file and set the `brokers`, `topic` and `saslJaasConfig` values to match your Kafka cluster
```

Comment:
I think this change should be separated out into a different PR as it is unrelated to the example you want to contribute. (separation of concerns)

Reply:
thanks Ivo, appreciate your extra effort!